src/ilastik-packager.imjoy.html

```diff
@@ -0,0 +1,348 @@
+<config lang="json">
+{
+  "name": "ilastik-Packager",
```

Comment:
You might want to call it `ilastik`, since this will also be listed under `applications` which represent the entry for ilastik software, this will be the entry to other features in the future, e.g. implement the test run features.